ui/apps/platform/src/Components/CompoundSearchFilter/components/CheckboxSelect.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactElement } from 'react';
+import {
+    Select,
+    SelectOption,
+    SelectList,
+    MenuToggle,
+    MenuToggleElement,
+    Badge,
+} from '@patternfly/react-core';
+
+type CheckboxSelectProps = {
+    selection: string[];
+    onChange: (value: string[]) => void;
+    children: ReactElement<typeof SelectOption>[];
+    isDisabled?: boolean;
+    ariaLabelMenu?: string;
+    toggleLabel?: string;
+};
+
+function CheckboxSelect({
```

Comment:
I was uncertain whether to keep this as a separate component or merge it with the `SimpleSelect` component. My preference is to keep it separate to maintain clarity for the distinct use case.